task/sast-snyk-check/0.4/tests/test-sast-snyk-check-no-token.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-sast-snyk-check-no-token
+spec:
+  description: |
+    Test the sast-snyk-check task is skipped when no token is available
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: scan-with-snyk
+      workspaces:
+        - name: workspace
+          workspace: tests-workspace
+      taskRef:
+        name: sast-snyk-check
+      params:
+        - name: image-url
+          value: "https://github.com/konflux-ci/build-definitions.git"
```

Comment:
Although the test probably does not care, this does not look like a valid image URL.  Those usually look like this: `quay.io/redhat-user-workloads/kdudka-tenant/aap-konflux-pipelines@sha256:f672fb7e55e13b6874a86e42159d87b4d7cf3a2119a545b35d34ed66b07948f0`